packages/client/hmi-client/src/model-representation/mira/mira.ts

```diff
@@ -523,14 +523,15 @@ export const convertToIGraph = (
 				data: {}
 			});
 
-			t.controllers.forEach((controllerName) => {
-				graph.edges.push({
-					source: t.name,
-					target: controllerName,
-					points: [],
-					data: { isController: true }
-				});
-			});
+			// Turn off dual-controller edges as per request - Nov 2024
```

Comment:
I suspect ideally this is determined by a flag/toggle rather than just commenting it out

Reply:
I don't think we need this kind of flexibility, however we are still playing around - thus the comment instead of total removal.